services/workflows-service/src/metrics/metrics.controller.ts

```diff
@@ -110,4 +119,19 @@ export class MetricsController {
   async getWorkflowDefinitionVariantsMetric(@ProjectIds() projectIds: TProjectIds) {
     return await this.metricsService.getWorkflowDefinitionVariantsMetric(projectIds);
   }
+
+  @common.Get('home')
+  @ApiResponse({
+    status: 403,
+    description: 'Forbidden',
+    schema: Type.Record(Type.String(), Type.Unknown()),
+  })
+  @Validate({
+    response: HomeMetricsSchema,
+  })
+  async getHomeMetrics(
+    @ProjectIds() projectIds: TProjectIds,
```

Comment:
Usually we use this decorator.
```suggestion
    @CurrentProject() currentProjectId: TProjectId,
```